package.xml

```diff
@@ -13,6 +13,8 @@ SPDX-License-Identifier: Apache-2.0
   <description>Simple wrapper around SocketCAN</description>
   <maintainer email="ramon.wijnands@nobleo.nl">Ramon Wijnands</maintainer>
   <license>Apache-2.0</license>
+  <url type="bugtracker">https://github.com/nobleo/nobleo_socketcan_bridge/issues</url>
+  <url type="repository">https://github.com/nobleo/nobleo_socketcan_bridge</url>
```

Comment:
I would flip the order. I _think_ in the generation they just pick the first :slightly_smiling_face:

Reply:
Done